### PR TITLE
Add new parameter TARGET_UBUNTU_MIRROR to config

### PR DESCRIPTION
**Pull request by NotNorom**
I used the original script today not realizing I was using a us mirror from Germany. I wanted to make these changes myself but  @sostela nailed it already.

### CODE REVIEW

**Review by kgilmer — CHANGES_REQUESTED** [review]
The config script version needs to bump so that users with older config scripts will know to take the update with the mirror defined.  See https://github.com/mvallim/live-custom-ubuntu-from-scratch/blob/master/scripts/default_config.sh#L67

File: scripts/default_config.sh [hunk 1]
```diff
@@ -9,6 +9,10 @@
 # See https://wiki.ubuntu.com/DevelopmentCodeNames for details
 export TARGET_UBUNTU_VERSION="focal"
 
+# The Ubuntu Mirror URL. It's better to change for faster download.
+# More mirrors see: https://launchpad.net/ubuntu/+archivemirrors
+export  TARGET_UBUNTU_MIRROR="http://mirrors.aliyun.com/ubuntu/"
```
[the remark this answers is not in the document]
Comment (NotNorom) [reply scripts/default_config.sh:14]: You mean http://us.archive.ubuntu.com/ubuntu/ ?

[the remark this answers is not in the document]
Comment (kgilmer) [reply scripts/default_config.sh]: Yes, that's it, thanks.

**Comment by kgilmer** [comment]
@mvallim are you ok with this change?  I see no problems.

**Review by kgilmer — APPROVED** [review]
LGTM

**Comment by NotNorom** [comment]
I'm confused was to why the build shows failing

**Pull request merged**